lib/browsersync.js

```diff
@@ -9,8 +9,9 @@ var bsConfigDefaults = {
 
 module.exports = function (app) {
   var self = this;
-  var bs = browsersync.create();
   var bsConfig = self.config.browsersync || {};
+  var bsInstanceName = bsConfig.instanceName || '';
```

Comment:
This may be a nitpick, but I'd prefer to keep `undefined`  as the default value instead of the empty string. Their doc mentions a 0-arg signature and a 1-string arg signature but does not say that `""` is a magic value and can't be used as a name.

Both have the same effect today, but it's [an undocumented implementation detail](https://github.com/BrowserSync/browser-sync/blob/ef0f947257416c593f88dbce11cd3a24f0fadcab/lib/index.js#L332).